### PR TITLE
Gym overview

### LINKED ISSUE #8: As a gym user, I want to find the different gyms, so that I can select a gym to view its information.
- [ ] I want to scroll through a list of all the available gyms 
- [ ] I want to press a button that provides information about the gym

### PR DESCRIPTION
**Pull request by chronis98**
Plz review my code

### CODE REVIEW

File: frontend/src/views/AddEquipment.vue [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        <div class="Details"> Select Equipment</div>
+        <button class="button" @click="isDropdownOpen=1">Test</button>
+            <!-- <div v-if="value.isDropdownOpenclass==1"></div> -->
+            <div v-for=" e,index in gymEquipment">
+                <div class='card' id="card{{index}}">
+                    
+                    <img src="@/assets/equipment.jpeg" />
+    
+                    
+                    <!-- <div class="right">
```
Comment (ArieBisfki) [inline frontend/src/views/AddEquipment.vue]: remove comment

File: frontend/src/views/AddEquipment.vue [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    
+            </div>
+    
+
+
+    
+    </template>
+    
+    <!-- <style>
+    @media (min-width: 1024px) {
```
Comment (ArieBisfki) [inline frontend/src/views/AddEquipment.vue]: remove comments

**Review by ArieBisfki — CHANGES_REQUESTED** [review]

File: api/dist/index.js
Comment (ArieBisfki) [inline api/dist/index.js:1]: remove dist folder and add it to gitignore

**Review by AbdulZor — APPROVED** [review]

File: api/tsconfig.json
Comment (ArieBisfki) [inline api/tsconfig.json:1]: discard change

File: frontend/src/views/CardComponent.vue [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<script  lang="ts">
+import { defineComponent } from "vue";
+export default defineComponent({
```
Comment (ArieBisfki) [inline frontend/src/views/CardComponent.vue]: whitespace

File: frontend/src/views/GymDetails.vue [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+import router, { RouteName } from "@/router";
+import CardComponent from '../views/CardComponent.vue';
+
+
+export type Location = {
+  name: string;
+  date: string;
+  numberOfPeople: number;
+  complete: boolean;
+};
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue:45]: leave whitespace after Location type

File: frontend/src/views/GymDetails.vue [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+</template>
+
+
+<script lang="ts">
+
+import { defineComponent } from "vue";
+import { useRoute, useRouter } from "vue-router";
+import router, { RouteName } from "@/router";
+import CardComponent from '../views/CardComponent.vue';
+
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: make whitespace only 1 line

File: frontend/src/views/GymDetails.vue [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+          "Power rack",
+          "Chest fly machine",
+          "Bench Press",
+          "Incline bench press"
+        ]
+      }
+    ];
+    
+    const gymName = route.params.name as string;
+    const gym = locations.find(location => location.name === gymName)!.name;
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: remove !.name

File: frontend/src/views/GymOverview.vue [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+import GymDetails from '../views/GymDetails.vue';
+import { defineComponent } from "vue";
+import type { Location } from "@/views/GymDetails.vue";
+import { useRouter } from "vue-router";
+import { RouteName } from "@/router";
+import HelloWorld from '../components/HelloWorld.vue'
+import CardComponent from '../views/CardComponent.vue';
+
+export default defineComponent({
+	components: {
```
Comment (ArieBisfki) [inline frontend/src/views/GymOverview.vue]: fix indent

File: frontend/src/views/GymOverview.vue [hunk 8]
```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="Title">Gym Overview</div>
```
Comment (ArieBisfki) [inline frontend/src/views/GymOverview.vue]: fix indent

File: frontend/src/views/GymOverview.vue [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+			locations,
+			handleClick
+		};
+	}
+});
+
+</script>
+
+<style scoped lang="scss">
+.Title {
```
Comment (ArieBisfki) [inline frontend/src/views/GymOverview.vue]: .title

File: frontend/src/views/GymOverview.vue [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+img {
+	width: 50px;
+	height: 50px;
+	display: absolute;
+	border-radius: 10%;
+}
+
+.gymTitle {
```
Comment (ArieBisfki) [inline frontend/src/views/GymOverview.vue]: .gym-title

**Review by ArieBisfki — CHANGES_REQUESTED** [review]

File: frontend/src/views/GymDetails.vue [hunk 11]
```diff
@@ -0,0 +1,312 @@
...
+  display: absolute;
+  border-radius: 10%;
+}
+
+
+.card:hover,
+.plus:hover {
+  box-shadow: 0px 10px 16px 0 rgba(0, 0, 0, 0.901);
+  cursor: pointer;
+
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: remove whitespace

File: frontend/src/views/GymDetails.vue [hunk 12]
```diff
@@ -0,0 +1,312 @@
...
+}
+
+
+
+.Title {
+  font-style: 'large';
+  font-size: 36.56px;
+  color: #FDFDFD;
+  margin: 5px;
+
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: remove whitespace

File: frontend/src/views/GymDetails.vue [hunk 13]
```diff
@@ -0,0 +1,312 @@
...
+  },
+})
+
+</script>
+<style scoped lang = "scss">
+#app {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: remove whitespace

File: frontend/src/views/GymDetails.vue [hunk 14]
```diff
@@ -0,0 +1,312 @@
...
+  font-weight: normal;
+}
+
+
+
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: only one line of whitespace

File: frontend/src/views/GymDetails.vue [hunk 15]
```diff
@@ -0,0 +1,312 @@
...
+  margin: 0 25px 0 0;
+  right: -8px;
+  --b: 4px;
+  width: 38px;
+  aspect-ratio: 1;
+  border: 10px solid #0375F7;
+  border-radius: 50%;
+  background:
+    conic-gradient(from 90deg at var(--b) var(--b), #0375F7 90deg, #fff 0) calc(100% + var(--b)/2) calc(100% + var(--b)/2)/ calc(50% + var(--b)) calc(50% + var(--b));
+
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: remove whitespace

**Review by ArieBisfki — CHANGES_REQUESTED** [review]

File: frontend/src/views/Card.vue [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+		{{ name }}<br>
+		<slot>
+
+		</slot>
+	</div>
+</template>
+
+<style scoped lang="scss">
+
+@media (hover: hover) {
```
Comment (ArieBisfki) [inline frontend/src/views/Card.vue]: no need

File: frontend/src/views/Card.vue [hunk 17]
```diff
@@ -0,0 +1,79 @@
+<script  lang="ts">
+
+import { defineComponent } from "vue";
+export default defineComponent({
+	name: "Card",
```
Comment (ArieBisfki) [inline frontend/src/views/Card.vue]: reduce indent

File: frontend/src/views/GymDetails.vue [hunk 18]
```diff
@@ -0,0 +1,287 @@
...
+export type Location = {
+  name: string;
+  equipment: string[];
+  date: string;
+  numberOfPeople: number;
+  complete: boolean;
+};
+
+export type Equipment = {
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: remove unused type

File: frontend/src/views/GymDetails.vue [hunk 19]
```diff
@@ -0,0 +1,287 @@
...
+    
+    const card = document.getElementById("test");
+    console.log(card);
+    function addEquipment(name: string): void {
+    };
+
+    return {
+      gymName,
+      locations,
+      gymEquipment: gym?.equipment,
```
Comment (ArieBisfki) [inline frontend/src/views/GymDetails.vue]: gym!.equipment

File: frontend/src/views/GymOverview.vue [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+        ], date: "10/02/2023", numberOfPeople: 12, complete: false },
+      { name: "Get-Fit Den Haag",equipment: [
+          "Power rack",
+          "Chest fly machine",
+          "Bench Press",
+          "Incline bench press"
+        ], date: "04/02/2020", numberOfPeople: 2, complete: true }
+    ];
+
+		const router = useRouter();
```
Comment (ArieBisfki) [inline frontend/src/views/GymOverview.vue]: fix indent

**Review by ArieBisfki — CHANGES_REQUESTED** [review]

**Review by ArieBisfki — APPROVED** [review]

**Pull request merged**